_targets/build.project.armv7m7-imxrt117x

```diff
@@ -70,8 +74,8 @@ PREINIT_SCRIPT=(
 USER_SCRIPT=(
 	"kernel ${BOOT_DEVICE}"
 	"app ${BOOT_DEVICE} -x dummyfs xip1 ocram2"
-	"app ${BOOT_DEVICE} -x imxrt-multi xip1 ocram2"
-	"app ${BOOT_DEVICE} -x psh xip1 ocram2"
+	"app ${BOOT_DEVICE} -x imxrt-multi xip1 ocram2;aips14;aips5"
+	"app ${BOOT_DEVICE} -x psh xip1 ocram2;flexspi"
```

Comment:
Why `psh` needs FlexSPI TX/RX FIFO memory range for data map ?

Reply:
It does not, removed